### DISCUSSION TITLE
how to use new version  minemacs-ng

### DISCUSSION THREAD

**Discussion by donneyluck** [abougouffa/minemacs#175] [Q&A]
I've switched to a new version of minemacs but don't know how to use it
evil-mode is now deprecated
Most people use hydra or meow mode to edit
So you need a usage document to help  get started quickly.

**Comment by abougouffa** [comment]
Well the new version is just Emacs! There is a lot of documentation about how to use Emacs with the default keybindings

**Reply by donneyluck** [reply]
Okay, I see what you mean. I still think a lot of key combinations    Ctrl+    Meta+    Work efficiency is very low

**Reply by donneyluck** [reply]
I think I might have to get used to it.

**Reply by abougouffa** [reply]
You can use Evil or Meow (both modules are obsolete, but still you can enable them and they will work out of the box)

### Edit
By the way, the builtin `viper-mode` gives a decent Vi-like modal editing experience in Emacs, it ia not as extensive as Evil mode, but it is a very good alternative and a very fast and integrated package.
